data/formatters/windows.yaml

```diff
@@ -78,6 +78,26 @@ message:
 - '[Identifier: {task_identifier}]'
 short_message:
 - 'Task: {task_name}'
+---                                                                                
+type: 'conditional'                                                                
+data_type: 'windows:diagnosis:eventtranscript'                                     
+message:                                                                           
+- 'full_event_name: {full_event_name}'                                             
+- 'logging_binary_name: {logging_binary_name}'                                     
+- 'friendly_logging_binary_name: {friendly_logging_binary_name}'                   
+- 'ver: {ver}'                                                                     
```

Comment:
I'll update these in bulk, but leaving comments so you know what changed for future reviews.
ver => version